config/manifests/bases/koku-metrics-operator.clusterserviceversion.yaml

```
@@ -18,6 +18,11 @@ metadata:
     support: Cost Management
   name: koku-metrics-operator.v0.0.0
   namespace: placeholder
+  labels:
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.ppc64le: supported
```

```suggestion
    operatorframework.io/arch.arm64: supported
    operatorframework.io/arch.ppc64le: supported
```

Let's also add arm64 here.